frontend/src/views/Listen.vue

```diff
@@ -0,0 +1,68 @@
+/*
+ * sri sri guru gauranga jayatah
+ */
+<template>
+  <div>
+    <v-toolbar :clipped-left="$vuetify.breakpoint.lgAndUp" fixed app>
+      <v-toolbar-title>
+        <router-link to="/" tag="span" style="cursor: pointer">
+          <h3>Audio File Name: {{ fileName ? fileName.split('.')[0]: null }}</h3>
```

Comment:
It's not good to put this logic into the template. Rather create another computed property for the base name of the file.